ner_v1/detectors/pattern/phone_number/phone_detection.py

```diff
@@ -126,7 +141,8 @@ def _detect_phone_format(self, phone_list=None, original_list=None):
             original_list.append(original)
         return phone_list, original_list
 
-    def mobile_number_regx(self, text):
+    @staticmethod
+    def mobile_number_regx(text):
```

Comment:
No need to make this function static
In fact, since this is an internal function, I would suggest renaming this to `_ detect_mobile_number_pattern` or something similar

Reply:
I know this is not in scope, but the body of this detector covers only Indian numbers and that too need 91 to match. We should definitely switch to more generic phone number patterns